platforms/java/versionsToBuild.txt

```diff
@@ -1,10 +1,11 @@
-# version, build number, sha
+# version, buildNumber, sha, jdkUrl, rootDirName
 1.8.0, b01, 1285da6278f2d38a790a21148d7e683f20de0799c44b937043830ef6b57f58c4
```

Comment:
Just to confirm: we're introducing SHA validation for Java when building the binaries, but do we use the SHA anywhere else? I'm curious why we'd be providing them in this file if they were (potentially) not use previously.

Reply:
No, there were no traces found in the Oryx repo for Java specifically.